packages/daimo-contract/src/shovel/config.json

```diff
@@ -0,0 +1,1110 @@
+            "type": "bytea"
+          }
+        ]
+      },
+      "block": [
+        {
+          "name": "chain_id",
+          "column": "chain_id"
+        },
+        {
+          "name": "block_num",
+          "column": "block_num"
+        },
+        {
+          "name": "block_hash",
+          "column": "block_hash"
+        },
+        {
+          "name": "tx_idx",
+          "column": "tx_idx"
+        },
+        {
+          "name": "tx_hash",
+          "column": "tx_hash"
+        },
+        {
+          "name": "log_addr",
+          "column": "log_addr"
+        }
+      ],
+      "event": {
+        "name": "SigningKeyRemoved",
+        "type": "event",
+        "anonymous": false,
+        "inputs": [
+          {
+            "indexed": true,
+            "name": "account",
+            "type": "address",
+            "column": "account"
+          },
+          {
+            "indexed": false,
+            "name": "keySlot",
+            "type": "uint8",
+            "column": "key_slot"
+          },
+          {
+            "indexed": false,
+            "name": "key",
+            "type": "bytes32[2]",
+            "column": "key"
+          }
+        ]
+      }
+    },
+    {
+      "name": "transfers",
+      "enabled": true,
+      "sources": [
+        {
+          "name": "$CHAIN_NAME",
+          "start": "$CHAIN_START_BLOCK"
+        }
+      ],
+      "table": {
+        "name": "transfers",
+        "columns": [
+          {
+            "name": "chain_id",
+            "type": "numeric"
+          },
+          {
+            "name": "block_num",
+            "type": "numeric"
+          },
+          {
+            "name": "block_hash",
+            "type": "bytea"
+          },
+          {
+            "name": "tx_idx",
+            "type": "numeric"
+          },
+          {
+            "name": "tx_hash",
+            "type": "bytea"
+          },
+          {
+            "name": "log_addr",
+            "type": "bytea"
+          },
+          {
+            "name": "f",
+            "type": "bytea"
+          },
+          {
+            "name": "t",
+            "type": "bytea"
+          },
+          {
+            "name": "v",
+            "type": "numeric"
+          }
+        ]
+      },
+      "block": [
+        {
+          "name": "chain_id",
+          "column": "chain_id"
+        },
+        {
+          "name": "block_num",
+          "column": "block_num"
+        },
+        {
+          "name": "block_hash",
+          "column": "block_hash"
+        },
+        {
+          "name": "tx_idx",
+          "column": "tx_idx"
+        },
+        {
+          "name": "tx_hash",
+          "column": "tx_hash"
+        },
+        {
+          "name": "log_addr",
+          "column": "log_addr",
+          "filter_op": "contains",
+          "filter_arg": [
+            "0x833589fCD6eDb6E08f4c7C32D4f71b54bdA02913",
+            "0x036CbD53842c5426634e7929541eC2318f3dCF7e"
```

Comment:
This is Base USDC + Base Sepolia USDC

If we're parameterizing by `$CHAIN_NAME` etc we might want to have a `$COIN_ADDR` too

Reply:
will leave for future